### PR TITLE
Adding Chandra A and Chandra O to Baraha for Devanagari (for Marathi)

### PR DESCRIPTION
**Pull request by abhagwat**
This change lets one write the characters ॲ, ऑ, कॅ, कॉ in the Baraha input scheme. These characters are essential for writing Marathi in a modern setting.

### CODE REVIEW

**Comment by ratreya** [comment]
I was proposing to create a separate language for Marathi: https://www.facebook.com/groups/lipika.ime/permalink/1934586116807202/. We would just override the following. Then it would work for all schemes not just Baraha. Thoughts? 
DEPENDENT SHORT E 090D (ऍ)
DEPENDENT SHORT O 0911 (ऑ)
VOWEL SHORT E 0945 (ॅ)
VOWEL SHORT O 0949 (ॉ)

**Comment by abhagwat** [comment]
I suppose that should work. (It definitely takes care of my use case!) I don't think anyone uses ऎ/ऒ in native Marathi writing anyway - I've never seen it in my life. As far as I can tell (from a bit of Googling), ऎ/ऒ are used exclusively to transcribe Kannada/Telugu to Devanagari, since these scripts apparently distinguish between short e/o and long e/o. Marathi writing doesn't have this distinction.

That said, I agree with your sentiment in the Facebook post you've linked. From a 'puritan perspective', we shouldn't have separate profiles for three languages written in Devanagari. As a matter of fact, ऑ is not even confined to just Marathi - even Hindi uses it (see, e.g. https://hi.wikipedia.org/wiki/%E0%A4%B9%E0%A5%88%E0%A4%B0%E0%A5%80_%E0%A4%AA%E0%A5%89%E0%A4%9F%E0%A4%B0). Based on this, I have two suggestions -
1) Implement the 4 overrides you've written above to Devanagari as a whole.
2) Incorporate the CANDRA A, CANDRA E and CANDRA O characters into Devanagari.map (as in my pull request). Baraha has keymappings for these, and I don't think it's necessarily a huge problem if these characters are not accessible from other schemes (since they do not have mappings for them).

By the way, the character that Marathi actually uses is 0972 (ॲ), not 090D (ऍ). I don't know why the latter was ever given a unicode character (I've never seen it except in unicode text).

File: Common/Resources/Schemes/Script/Devanagari.map [hunk 1]
```diff
@@ -94,10 +96,12 @@ VOWEL	VOCALIC L	090C
 VOWEL	SHORT E	090E
 VOWEL	E	090F
 VOWEL	AI	0910
+VOWEL	CANDRA O	0911
 VOWEL	SHORT O	0912
 VOWEL	O	0913
 VOWEL	AU	0914
 VOWEL	VOCALIC RR	0960
 VOWEL	VOCALIC LL	0961
+VOWEL	CANDRA A	0972
```
Comment (ratreya) [inline Common/Resources/Schemes/Script/Devanagari.map]: Shouldn't this be called CANDRA E?

File: Common/Resources/Schemes/Transliteration/Baraha.tlr [hunk 2]
```diff
@@ -95,8 +97,10 @@ VOWEL	VOCALIC L	~lu
 VOWEL	SHORT E	e
 VOWEL	E	E
 VOWEL	AI	ai
+VOWEL	CANDRA O	~o
 VOWEL	SHORT O	o
 VOWEL	O	O
 VOWEL	AU	au, ou
 VOWEL	VOCALIC RR	RU
 VOWEL	VOCALIC LL	~lU
+VOWEL	CANDRA A	~a
```
Comment (ratreya) [inline Common/Resources/Schemes/Transliteration/Baraha.tlr]: Shouldn't this be called CANDRA E?

File: Common/Resources/Schemes/Script/Devanagari.map (hunk 1 above)
Comment (abhagwat) [reply Common/Resources/Schemes/Script/Devanagari.map]: It's a bit weird, but CANDRA E is ऍ, and CANDRA A is ॲ (Marathi uses the latter), so my choice of CANDRA A was deliberate.
Also, it seems to me that the CANDRA variants are right above their respective SHORT variants (unless you meant something else).
Let me know if anything else needs to be changed!

**Comment by ratreya** [comment]
Thanks!

**Pull request merged**